src/libs/ajax/data-table-providers/WdsDataTableProvider.ts

```diff
@@ -105,17 +105,18 @@ export class WdsDataTableProvider implements DataTableProvider {
   tsvFeatures: TSVFeatures = {
     needsTypeInput: true,
     sampleTSVLink: 'https://storage.googleapis.com/terra-featured-workspaces/Table_templates/template_sample-wds-table.tsv', //TODO: This location may need to change
-    invalidFormatWarning: 'Invalid format: Data does not include sys_name column.',
-    isInvalid: (options: InvalidTsvOptions): boolean => {
-      return options.fileImportModeMatches && !options.sysNamePresent && options.filePresent
+    isInvalid: (): boolean => {
+      // WDS does not have any restrictions on what can be uploaded, as entity_id
+      // as  is not required like in Entity Service for GCP.
+      return false
```

Comment:
I decided to keep the value here as a default, rather than removing entirely.

The reason being that removing entirely would result in a decent amount of logic/refactoring in CSS logic for WDS. This value is quite benign now, as it is not included in any downstream functions such as `disabled()` and `tooltip()`